CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/MainActivity.kt

```diff
@@ -75,4 +77,17 @@ class MainActivity : AppCompatActivity() {
                 mediaDir else appContext.filesDir
         }
     }
+
+    private fun hideSystemUI() {
+        WindowCompat.setDecorFitsSystemWindows(window, false)
+        WindowInsetsControllerCompat(window, container).let { controller ->
+            controller.hide(WindowInsetsCompat.Type.systemBars())
+            controller.systemBarsBehavior = WindowInsetsControllerCompat.BEHAVIOR_SHOW_TRANSIENT_BARS_BY_SWIPE
+        }
+    }
+
+    private fun showSystemUI() {
```

Comment:
This method is unused, I'm not sure if it's necessary? Upon leaving the app, the system should figure out its own state.

Reply:
yeah, thought to leave it here and maybe for future reference.

Reply:
My preference would probably be to remove something that's unused, otherwise it might look like a bug that there's a method that is never called. Don't feel too strongly though!

Reply:
make sense. removed it